serious_stuff/lib/components/Fab.dart

```diff
@@ -1,11 +1,17 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_speed_dial/flutter_speed_dial.dart';
 import 'package:serious_stuff/constants.dart' as Constants;
+import 'package:serious_stuff/screens/SettingsPage.dart' as Settings;
+
+SpeedDial buildOptionsFAB(BuildContext context, bool isMenuLeft) {
+  double currentPos = 18;
+  if (isMenuLeft) {
+    currentPos = MediaQuery.of(context).size.width - 80;
+  }
```

Comment:
We seem to have some magic numbers here, are we hard coding the position of the button? Will this work on different screens? i.e. phone vs tablet.